js/main.js

```diff
@@ -1,128 +1,197 @@
-(function(){
-
-  'use strict';
-
+(function() {
+  "use strict";
   $(document).ready(initialize);
 
-  function initialize(){
-
-  }
+  function initialize() {}
 
-  var datasetButtons= ['#fire-station','#police-station','#wifi', '#parks','#community'];
+  var datasetButtons = [
+    "#fire-station",
+    "#police-station",
+    "#wifi",
+    "#parks",
+    "#community"
+  ];
 
-  datasetButtons.forEach(function(dataset){
-    $(dataset).click(function(){
+  datasetButtons.forEach(function(dataset) {
+    $(dataset).click(function() {
       //add toggle for showing icons on map
-      toggleIcon($(this).attr('id'));
+      toggleIcon($(this).attr("id"));
     });
   });
 
-
-/*
+  /*
   The boundaries for Nashville are
   upper-left: [35.9758, -86.9815]
   lower-right: [36.3350, -86.4542]
 
   This center and zoom doesn't capture the entire box, but includes
   all of the points in our datasets without a lot of blank space.
 */
-  var map = L.map('map').setView(
+  var map = L.map("map").setView(
     // center and zoom
-    [36.165818, -86.784245], 12
-  )
+    [36.165818, -86.784245],
+    12
+  );
 
   //varables for the map. points are latitude and longitude points
   //markers are leaflet markers
-  var firePoints=[], parksPoints=[], policePoints=[], wifiPoints=[], communityPoints=[],
-    fireMarkers=[], policeMarkers=[], wifiMarkers=[], parksMarkers=[], communityMarkers=[]
+  var firePoints = [],
+    parksPoints = [],
+    policePoints = [],
+    wifiPoints = [],
+    communityPoints = [],
+    fireMarkers = [],
+    policeMarkers = [],
+    wifiMarkers = [],
+    parksMarkers = [],
+    communityMarkers = [];
   //set up the icons for the markers
-  var fireIcon = new L.Icon({iconUrl:'../images/firestation-color.png',
-                            iconSize:[45,45]}),
-      policeIcon = new L.Icon({iconUrl:'../images/policestation-color.png',
-                              iconSize:[45,45]}),
-      wifiIcon = new L.Icon({iconUrl:'../images/wifi-color.png',iconSize:[45,45]}),
-      parksIcon = new L.Icon({iconUrl:'../images/park-color.png',iconSize:[45,45]}),
-      communityIcon = new L.Icon({iconUrl:'../images/home-color.png',iconSize:[45,45]});
+  var fireIcon = new L.Icon({
+    iconUrl: "../images/firestation-color.png",
```

Comment:
Nit: Make this line and the next two for fireIcon have the same indent as the other icon variables.